pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -161,6 +161,18 @@ def advance(self, *args: Any, **kwargs: Any) -> None:
             # indicate the loop has run
             self._has_run = True
 
+    def on_advance_start(self) -> None:  # type: ignore[override]
```

Comment:
to avoid the 
`# type: ignore[override]`
you can simply add the args kwargs

Reply:
Done! Thanks for the suggestion, @awaelchli.

Reply:
Should be fixed now :)